### PR TITLE
Adding Specifically Clementines to Recipe section

### PR DESCRIPTION
**Pull request by davideshay**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [X ] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [X] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation
  such as title case, all caps, small caps or all lowercase.
  `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [ X] Additions that depend on proprietary services outside the user's control must be marked `⚠`.
  ``- [Name](http://homepage/) `⚠` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [X] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software) must be added to `non-free.md` and have the license set to `⊘ Proprietary`
- [X] Additions are inserted preserving alphabetical order.
- [X] Additions are not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [X] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [X] Any license you add is in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [X] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [X] Any category you are creating has the minimum requirement of 3 items.
  If not, your addition may be inserted into `Misc/Other`.
- [X] Any software project you are adding to the list is actively maintained.
- [X] Any software project you are adding has working installation instructions.
- [X] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

### CODE REVIEW

**Review by KieranRobson — CHANGES_REQUESTED** [review]
> https://davideshay.github.io/groceries/ 

The `Installation Guide` option on the side bar doesnt work and I could not find any other way to get to it, without this, there is no way for the average user to find out how to install. Once this gets fixed, I'd be happy to take another look.

**Comment by davideshay** [comment]
The "Installation Guide" on the left-hand side is a heading and you can't click on it. When I look at it, underneath that header is "Key Concepts", "Installation" and "Docker Instructions". Do you see those? If you click on those do you see the relevant info?

**Comment by KieranRobson** [comment]
> The "Installation Guide" on the left-hand side is a heading and you can't click on it. When I look at it, underneath that header is "Key Concepts", "Installation" and "Docker Instructions". Do you see those? If you click on those do you see the relevant info?

My mistake! Ill carry on with this review!

File: README.md [hunk 1]
```diff
@@ -1673,6 +1673,7 @@ Software and tools for managing [recipes](https://en.wikipedia.org/wiki/Recipe).
 - [Recepturer](https://recepturer.com/) `⚠` - Let Recepturer help you organize all your recipes in one place. You can add, edit and delete recipes, and create meal plans. Simply type your recipe and all ingredients will be listed as you go. Data is stored on Dropbox. ([Source Code](https://github.com/sjoerdvanderhoorn/recepturer)) `MIT` `Javascript`
 - [RecipeSage](https://github.com/julianpoy/recipesage) - A recipe keeper, meal plan organizer, and shopping list manager that can import recipes directly from any URL. ([Demo](https://recipesage.com)) `AGPL-3.0` `Nodejs`
 - [reciphpes!](https://github.com/nanawel/reciphpes) - A lightweight, Symfony-based recipes indexing software supporting search and tags, using a SQLite database. `MIT` `Docker`
+- [Specifically Clementines](https://davideshay.github.io/groceries/) - Grocery shopping app previously called "groceries". Reliable sync with multiple users/devices (web/mobile/Android/iOS). Includes recipes and integration with Tandoor. ([Source Code](https://github.com/davideshay/groceries)) `MIT` `JavaScript`
```
Comment (KieranRobson) [inline README.md:1676]:
```suggestion
- [Specifically Clementines](https://davideshay.github.io/groceries/) - Grocery shopping app previously called "groceries". Reliable sync with multiple users/devices (web/mobile/Android/iOS). Includes recipes and integration with Tandoor. ([Source Code](https://github.com/davideshay/groceries)) `MIT` `Docker/Nodejs`
```

> In the client directory, do npm install.
> Install via Docker

Please change the end tag to `Nodejss/Docker`

**Review by KieranRobson — CHANGES_REQUESTED** [review]
Quick small change to be made

**Comment by nodiscc** [comment]
Sorry, this guideline was added in the mean time (https://github.com/awesome-selfhosted/awesome-selfhosted/pull/3999):

> Any software project you are adding was first released more than 4 months ago.

Please resubmit when the first release is at least 4 months old.

**Comment by davideshay** [comment]
First release was 0.3.0 on April 1. Does this not qualify?

**Pull request closed without merging**